python/pyspark/ml/image.py

```diff
@@ -71,9 +88,30 @@ def ocvTypes(self):
         """
 
         if self._ocvTypes is None:
-            ctx = SparkContext._active_spark_context
-            self._ocvTypes = dict(ctx._jvm.org.apache.spark.ml.image.ImageSchema.javaOcvTypes())
-        return self._ocvTypes
+            ctx = SparkContext.getOrCreate()
+            ocvTypeList = ctx._jvm.org.apache.spark.ml.image.ImageSchema.javaOcvTypes()
+            self._ocvTypes = [self._OcvType(name=x.name(),
+                                            mode=x.mode(),
+                                            nChannels=x.nChannels(),
+                                            dataType=x.dataType(),
+                                            nptype=self._ocvToNumpyMap[x.dataType()])
+                              for x in ocvTypeList]
+        return self._ocvTypes[:]
```

Comment:
Is it for copy? I usually do `list(self._ocvTypes)` tho.

Reply:
yes it is for copy.